.migration-backup/ui-components/src/components/index.ts

```
@@ -12,4 +12,8 @@ export { LogoIcon } from './LogoIcon';
 export * from './atoms';
 export * from './molecules';
 export * from './organisms';
-export * from './templates'; 
\ No newline at end of file
+export * from './templates';
+
+// MIGRATION COMPATIBILITY: Re-export from new UI structure
+// This allows both old and new import paths to work during migration
+export * from '../ui';
```

ESLint

[error] Multiple exports of name 'Avatar'.

(import/export)